src/components/Introduction.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import styled, {keyframes} from 'styled-components';
+import Photo from "../images/pic-01.png";
```

Comment:
Please specify what `Photo` named for ? such as `SalweenPhoto`, `HouseImage` etc...